addon/mixins/translate3d-mixin.js

```diff
@@ -59,8 +57,10 @@ export default Mixin.create({
       // Wait while CSS takes affect
       // Set the `main` styles and run the transition-in styles
       run.next(() => {
-        this.waitTransitionEnd(this.element).then(() => {
-          this.onTranslateFromEnd();
+        this.waitTransitionEnd($(this.element)).then(() => {
+          if (!this.get('isDestroying') && !this.get('isDestroyed')) {
```

Comment:
What is the reason for this guard?
For situations where you create and destroy very quickly?
What was the problem you were having?